src/framecontainer/FrameContainer.ts

```diff
@@ -0,0 +1,126 @@
+import {createIframe} from './createIframe';
+import {appendParametersToURL} from './viewerParameters';
+import * as config from './config';
+import {Messaging} from '@ampproject/viewer-messaging';
+import renderingModules from './rendering-modules/index';
+
+export class FrameContainer {
```

Comment:
@fileoverview pls and documentation on the methods as well. Similar to the changes you made in https://github.com/ampproject/amp-email-viewer/commit/6871d7cbc1190973d81e03221304fad0246b245a